src/style.css

```diff
@@ -1,3 +1,104 @@
 body {
+  --header-height: 60px;
+
   margin: 0;
+  background-color: #00000029;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
+}
+
+header {
+  background-color: #fff;
+  width: 100vw;
+  height: var(--header-height);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  border: 2px solid transparent;
+}
+
+.img_link {
+  padding-left: 50px;
+}
+
+.nav_list {
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  padding: 0 50px 0 0;
+  align-items: center;
+}
+
+/* .nav_first_item{
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+
+  justify-content: space-between;
+} */
+
+/* .nav_1{
+  margin-bottom: 20px;
+} */
```

Comment:
There are commented-out styles that might be unnecessary if they are not intended for future use. Consider removing them to keep the CSS clean and maintainable.